### ISSUE TITLE
Support 91-1 EDIFACT

### ISSUE THREAD

**Issue by lawrence-tierney-daiwasports** [smooks/smooks-edi-cartridge#321]
Hi there,

It would be nice if Smooks had bindings and mappings for EDIFACT 91-1.

Reason: my organisation has a "black box" EDIFACT system for Invoices, Orders and Dispatch Advice, we produce and consume v91-1 messages. We are updating our DB and the system requires to be re-written from the ground up but it needs to support 91-1.

After having looked at the below realise this is unlikely.

https://github.com/smooks/smooks-edi-cartridge/issues/45
https://groups.google.com/g/smooks-user/c/sYrgs_FtHZs

Thanks in advance and keep up the good work!

**Comment by cjmamo** [comment]
I'd say it's not high in the priority list. If your organisation is open to funding its development then I'm sure that we can prioritise it.

**Comment by lawrence-tierney-daiwasports** [comment]
Yeah I figured as much; it is pretty ancient. TBH I can't see us funding the development (not that I personally would be adverse). Thanks for the response anyway.

**Comment by cjmamo** [comment]
You might want to consider using the EDI cartridge with a rudimentary DFDL schema that captures all segments and data elements. But if you need validation to happen in Smooks then it's a different story.

**Comment by lawrence-tierney-daiwasports** [comment]
Good idea. I'll look into it. Cheers mate
